src/java.base/share/classes/java/net/Socket.java

```diff
@@ -661,7 +661,7 @@ public void connect(SocketAddress endpoint, int timeout) throws IOException {
             throw new IllegalArgumentException("Unsupported address type");
 
         if (epoint.isUnresolved()) {
-            var uhe = new UnknownHostException(epoint.getHostName() + " is unresolved");
+            var uhe = new UnknownHostException(epoint.getHostName());
```

Comment:
In this instance, I believe reverting to the same message as was thrown before and reconciling Socket with SocketImpl is the right call. The change to the exception message was somewhat gratuitous. Since it makes several tests fail, let's avoid that.

Reply:
Changed as requested.